rules_ruby_gapic/ruby_gapic.bzl

```diff
@@ -16,83 +16,11 @@
 Rules for generating ruby code with protoc
 including rules for ruby, grpc-ruby, and various flavors of gapic-generator-ruby
 """
-load("@com_google_api_codegen//rules_gapic:gapic.bzl", "proto_custom_library", "GapicInfo")
-
-##
-# An implementation of the _ruby_gapic_library_add_gapicinfo
-#
-def _ruby_gapic_library_add_gapicinfo_impl(ctx):
-  return [
-    DefaultInfo(files = depset(direct = [ctx.file.output])),
-    GapicInfo(),
-  ]
-
-##
-# A rule that adds the GapicInfo provider to the input
-# used to mark the result of the gapic-generator-ruby plugin
-# to distinguish it from the output of other plugins (e.g. grpc)
-#
-_ruby_gapic_library_add_gapicinfo = rule(
-  _ruby_gapic_library_add_gapicinfo_impl,
-  attrs = {
-    "output": attr.label(allow_single_file = True)
-  }
+load(
+  ":private/ruby_gapic_library.bzl",
+  _ruby_gapic_library = "ruby_gapic_library",
```

Comment:
To avoid sorts of confusions, please consider renaming `ruby_gapic_library` from the private subfolder to something like `ruby_gapic_library_internal` or something like that. Otherwise "greping" the directory may be really confusing.

Reply:
makes sense